eppo_client/client.py

```diff
@@ -74,12 +74,15 @@ def get_numeric_assignment(
         default: float,
         subject_attributes: Optional[SubjectAttributes] = None,
     ) -> float:
-        return self.get_assignment_variation(
-            subject_key,
-            flag_key,
-            default,
-            subject_attributes,
-            VariationType.NUMERIC,
+        # convert to float in case we get an int
```

Comment:
👍  Took me a moment to understand – I first thought you were going to rescue a mismatch (`getNumeric` for an `Integer` flag) which I didn't like. But what you're doing is ensuring that `3.0` (a valid variation for a numeric flag) is indeed returned as a float and not as the integer `3`.